Two punctuation changes

ievastakvile opened the pull request:
Line 4: unnecessary comma before "and"
Line 31: added comma after an introductory phrase

ievastakvile committed:
Two punctuation changes

Line 4: unnecessary comma before "and"
Line 31: added comma after an introductory phrase

siefkenj commented on src/chapters/chapter5.tex:
```diff
@@ -1,7 +1,7 @@
 
 Fundamentally, an integral is the result of chopping a region up into tiny
-pieces, adding all the pieces up again, and taking a limit as the size of
+pieces, adding all the pieces up again and taking a limit as the size of
```
It's a list, so it should have a comma before the and.

ievastakvile committed:
Update chapter5.tex

Sorry, Oxford commas are not my thing, I'll try to stop myself from imposing my personal preferences on your book in the future, it's just a hard thing to do
